feconf.py

```diff
@@ -951,11 +952,19 @@ def get_empty_ratings():
 HANDLER_TYPE_HTML = 'html'
 HANDLER_TYPE_JSON = 'json'
 
-# Following are the constants for the roles.
+# Following are the constants for the roleIds. RoleIds are human
+# readable roles with underscores instead of spaces.
+# To check the role updation from /admin#roles, set ADMIN_SHOW_UPDATE_ROLE
+# constant in Admin.js to true.
 ROLE_GUEST = 'GUEST'
```

Comment:
Yes, I think so. We should try to be consistent in the backend code in order to remove ambiguity so that developers are not confused.